src/ima.js

```diff
@@ -866,4 +866,9 @@ export default class Ima extends BasePlugin {
   }
 }
 
+declare var __VERSION__: string;
```

Comment:
I prefer to create an index.js file which will import the ima.js, register it and will expose whatever he wants.
It will separate plugin logic and exports.